docker/README.md

```diff
@@ -21,16 +21,55 @@ docker-compose up -d
 
 Copier le sample.env en .env et rajouter si besoin des valeurs nécessaires
 
-## exemple utilisation Dev : 
-> Le démarrage de tous les projets en simmultané utilise beaucoup de ressources réseau car les projets sont build au démarrage.
+## exemple utilisation Dev d'API : 
 
 ```sh
-docker-compose -f docker-compose.yaml -f docker-compose-dev.yaml build 
-docker-compose -f docker-compose.yaml -f docker-compose-dev.yaml up mon-pix
+docker-compose -f docker-compose.yaml -f docker-compose-dev-api.yaml build 
+docker-compose -f docker-compose.yaml -f docker-compose-dev-api.yaml up -d 
```

Comment:
les logs

```
api_1       | > pix-api@3.280.1 preinstall
api_1       | > npx check-engine
api_1       | 
api_1       | glob error [Error: EACCES: permission denied, scandir '/root/.npm/_logs'] {
api_1       |   errno: -13,
api_1       |   code: 'EACCES',
api_1       |   syscall: 'scandir',
api_1       |   path: '/root/.npm/_logs'
api_1       | }
api_1       | npm WARN logfile Error: EACCES: permission denied, scandir '/root/.npm/_logs'
api_1       | npm WARN logfile  error cleaning log files [Error: EACCES: permission denied, scandir '/root/.npm/_logs'] {
api_1       | npm WARN logfile   errno: -13,
api_1       | npm WARN logfile   code: 'EACCES',
api_1       | npm WARN logfile   syscall: 'scandir',
api_1       | npm WARN logfile   path: '/root/.npm/_logs'
api_1       | npm WARN logfile }
api_1       | npm ERR! code EACCES
api_1       | npm ERR! syscall mkdir
api_1       | npm ERR! path /root/.npm/_cacache/tmp
api_1       | npm ERR! errno EACCES
api_1       | npm ERR! 
api_1       | npm ERR! Your cache folder contains root-owned files, due to a bug in
api_1       | npm ERR! previous versions of npm which has since been addressed.
api_1       | npm ERR! 
api_1       | npm ERR! To permanently fix this problem, please run:
api_1       | npm ERR!   sudo chown -R 1000:1000 "/root/.npm"
api_1       | 
api_1       | npm ERR! A complete log of this run can be found in:
api_1       | npm ERR! code 1
api_1       | npm ERR! path /code
api_1       | npm ERR! command failed
api_1       | npm ERR! command sh -c npx check-engine
api_1       | 
api_1       | npm ERR! A complete log of this run can be found in:
api_1       | npm ERR!     /root/.npm/_logs/2022-12-08T09_46_40_462Z-debug-0.log
```

Reply:
Est-ce que tu peux pull mes dernières modifs et faire un 
`docker volume rm docker_api_node_modules`
puis 

`docker-compose -f docker-compose.yaml -f docker-compose-dev-api.yaml up ` et me dire si ça fonctionne ?

Reply:
Erreur à nouveau
```api_1       | > pix-api@3.280.1 preinstall
api_1       | > npx check-engine
api_1       | 
api_1       | glob error [Error: EACCES: permission denied, scandir '/root/.npm/_logs'] {
api_1       |   errno: -13,
api_1       |   code: 'EACCES',
api_1       |   syscall: 'scandir',
api_1       |   path: '/root/.npm/_logs'
api_1       | }
api_1       | npm WARN logfile Error: EACCES: permission denied, scandir '/root/.npm/_logs'
api_1       | npm WARN logfile  error cleaning log files [Error: EACCES: permission denied, scandir '/root/.npm/_logs'] {
api_1       | npm WARN logfile   errno: -13,
api_1       | npm WARN logfile   code: 'EACCES',
api_1       | npm WARN logfile   syscall: 'scandir',
api_1       | npm WARN logfile   path: '/root/.npm/_logs'
api_1       | npm WARN logfile }
api_1       | npm ERR! code EACCES
api_1       | npm ERR! syscall mkdir
api_1       | npm ERR! path /root/.npm/_cacache/tmp
api_1       | npm ERR! errno EACCES
```

Reply:
Nouveau test KO en suivant la description de PR
```shell
 docker-compose -f docker-compose.yaml -f docker-compose-dev-api.yaml -f docker-compose-dev-front.yaml up api orga
pix-api-postgres is up-to-date
pix-api-redis is up-to-date
Starting docker_api_1 ... done
Starting docker_orga_1 ... done
Attaching to docker_api_1, docker_orga_1
orga_1      | node:fs:585
orga_1      |   handleErrorFromBinding(ctx);
orga_1      |   ^
orga_1      | 
orga_1      | Error: EACCES: permission denied, open '/root/.config/configstore/ember-cli.json'
orga_1      | You don't have access to this file.
orga_1      | 
orga_1      |     at Object.openSync (node:fs:585:3)
orga_1      |     at Object.readFileSync (node:fs:453:35)
orga_1      |     at Configstore.get all [as all] (/code/node_modules/configstore/index.js:34:25)
orga_1      |     at Configstore.get (/code/node_modules/configstore/index.js:77:27)
orga_1      |     at clientId (/code/node_modules/ember-cli/lib/cli/index.js:49:24)
orga_1      |     at module.exports (/code/node_modules/ember-cli/lib/cli/index.js:105:13)
orga_1      |     at Object.<anonymous> (/code/node_modules/ember-cli/bin/ember:34:1)
orga_1      |     at Module._compile (node:internal/modules/cjs/loader:1103:14)
orga_1      |     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
orga_1      |     at Module.load (node:internal/modules/cjs/loader:981:32) {
orga_1      |   errno: -13,
orga_1      |   syscall: 'open',
orga_1      |   code: 'EACCES',
orga_1      |   path: '/root/.config/configstore/ember-cli.json'
orga_1      | }
docker_orga_1 exited with code 1
api_1       | npm ERR! code EACCES
api_1       | npm ERR! syscall unlink
api_1       | npm ERR! path /code/node_modules/.package-lock.json
api_1       | npm ERR! errno -13
api_1       | npm ERR! Error: EACCES: permission denied, unlink '/code/node_modules/.package-lock.json'
api_1       | npm ERR!  [Error: EACCES: permission denied, unlink '/code/node_modules/.package-lock.json'] {
api_1       | npm ERR!   errno: -13,
api_1       | npm ERR!   code: 'EACCES',
api_1       | npm ERR!   syscall: 'unlink',
api_1       | npm ERR!   path: '/code/node_modules/.package-lock.json'
api_1       | npm ERR! }
api_1       | npm ERR! 
api_1       | npm ERR! The operation was rejected by your operating system.
api_1       | npm ERR! It is likely you do not have the permissions to access this file as the current user
api_1       | npm ERR! 
api_1       | npm ERR! If you believe this might be a permissions issue, please double-check the
api_1       | npm ERR! permissions of the file and its containing directories, or try running
api_1       | npm ERR! the command again as root/Administrator.
api_1       | 
api_1       | npm ERR! A complete log of this run can be found in:
api_1       | npm ERR!     /home/node/.npm/_logs/2022-12-16T11_03_43_890Z-debug-0.log
docker_api_1 exited with code 243

```